### ISSUE TITLE
Error, just installed it

### ISSUE THREAD

**Issue by TadeoKondrak** [not-an-aardvark/lucky-commit#2]
```

fatal: ambiguous argument 'head': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
child_process.js:533
    throw err;
    ^

Error: Command failed: git --no-pager show -s --oneline --format="%cd" head
fatal: ambiguous argument 'head': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

    at checkExecSyncError (child_process.js:490:13)
    at Object.execSync (child_process.js:530:13)
    at exec (/usr/lib/node_modules/lucky-commit/index.js:5:35)
    at luckyCommit (/usr/lib/node_modules/lucky-commit/index.js:23:24)
    at Object.<anonymous> (/usr/lib/node_modules/lucky-commit/index.js:40:1)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
```

**Comment by not-an-aardvark** [comment]
What version of git are you using?

**Comment by TadeoKondrak** [comment]
```$ git --version
git version 2.7.4```

**Comment by not-an-aardvark** [comment]
Looks like this is because some versions of git only accept `HEAD` as the current ref, whereas others also accept `head`. I'm not sure about the details of when lowercase `head` is allowed, but it seems like using uppercase in lucky-commit would fix this issue.

**Comment by not-an-aardvark** [comment]
Published a fix in v0.2.3. You can use `npm install --global lucky-commit@latest` to upgrade.
